editoast/openapi.yaml

```diff
@@ -8677,30 +8677,6 @@ paths:
       summary: Serialize an infra to railjson
       tags:
       - infra
-  /infra/{id}/speed_limit_tags/:
-    get:
-      parameters:
-      - description: Infra id
-        in: path
-        name: id
-        required: true
-        schema:
-          type: integer
-      responses:
-        '200':
-          content:
-            application/json:
-              schema:
-                example:
```

Comment:
I noticed we lost some examples in the documentation. I believe `utoipa` supports adding examples, maybe you can try to add them back?

Reply:
Done.